### DISCUSSION TITLE
Welcome to OpenUtau Discussions!

### DISCUSSION THREAD

**Discussion by stakira** [stakira/OpenUtau#345] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cimafon** [comment]
Hello! Do you guys have a discord link?

**Reply by MaximusWolfie** [reply]
There should be a link in the readme.md

**Reply by cimafon** [reply]
Aaah thank you! Sorry I missed it 🙏

**Comment by NasserDiscovers** [comment]
does v2-v6 vbs work here

**Reply by Nexys4t** [reply]
If you mean Vocaloid, I'm pretty sure it doesn't unless you make illegal activity wich is not recommended.

**Comment by shigeruotake26oct-pixel** [comment]
使い始めでシンガーを選択し、音符を書こうとするとアプリが落ちて終了してしまいます。どうすればいいの？

**Comment by deltaVOCALOID09378** [comment]
I want to add a Thai VCCV, Myanmar, Laos, Malay, and Indonesian dictionary feature, along with a dictionary showing correct pronunciation.
